src/index.js

```diff
@@ -84,42 +89,43 @@ function startQtumProcess(reindex) {
 }
 
 async function startAPI() {
-  syncRouter.applyRoutes(server);
-  apiRouter.applyRoutes(server);
-
   server.get(/\/?.*/, restify.plugins.serveStatic({
     directory: path.join(__dirname, '../ui'),
     default: 'index.html',
+    maxAge: 0,
   }));
 
+  syncRouter.applyRoutes(apiServer);
+  apiRouter.applyRoutes(apiServer);
+
+  apiServer.listen(5555, () => {
+    logger.debug('Bodhi Api is running on http://127.0.0.1:5555.');
```

Comment:
API still running on 5555? not on 5000?

Reply:
@dwalintukan it is correct, app on 5000, api on 5555